pandas/conftest.py

```diff
@@ -111,6 +111,35 @@ def axis_series(request):
     return request.param
 
 
+@pytest.fixture
+def datetime_frame():
```

Comment:
can we clarify that this is datetime index with float values?

Reply:
E.g. rename it to `float_frame_dt_index`? That would involve changing a lot of files, but sure, can do

Reply:
Seems Jeff would prefer this to be done separately, so have reverted for now